documentation/Functions.md

```diff
@@ -159,9 +159,15 @@ installCobiGen()
 ***
 
 ### cobiGenJava <a name="cobiGenJava"></a>
+This function is used for code generation inside Java file i.e. Entity class in Java.
```

Comment:
I would prefer somethink like:
`This function executes CobiGen on Java files to generate code based on them (e.g.  create CRUD operations from an entity class).`